examples/gke-public-cluster-istio/main.tf

```diff
@@ -0,0 +1,91 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY A GKE PUBLIC CLUSTER IN GOOGLE CLOUD PLATFORM
+# This is an example of how to use the gke-cluster module to deploy a public Kubernetes cluster in GCP with a
+# Load Balancer in front of it.
+# ---------------------------------------------------------------------------------------------------------------------
+
+terraform {
+  # The modules used in this example have been updated with 0.12 syntax, additionally we depend on a bug fixed in
+  # version 0.12.7.
+  required_version = ">= 0.12.7"
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# PREPARE PROVIDERS
+# ---------------------------------------------------------------------------------------------------------------------
+
+provider "google" {
+  version = "~> 2.9.0"
+  project = var.project
+  region  = var.region
+}
+
+provider "google-beta" {
+  version = "~> 2.9.0"
+  project = var.project
+  region  = var.region
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY A PUBLIC CLUSTER IN GOOGLE CLOUD PLATFORM
+# ---------------------------------------------------------------------------------------------------------------------
+
+module "gke_cluster" {
+  # When using these modules in your own templates, you will need to use a Git URL with a ref attribute that pins you
+  # to a specific version of the modules, such as the following example:
+  # source = "github.com/gruntwork-io/terraform-google-gke.git//modules/gke-cluster?ref=v0.2.0"
+  source = "../../modules/gke-cluster"
+
+  name = var.cluster_name
+
+  project  = var.project
+  location = var.location
+
+  # enable istio
+  enable_istio = var.enable_istio
```

Comment:
You can probably hard code this to `true` for this example.

Reply:
fixed + removed `enable_istio` from variables in istio example.